### DISCUSSION TITLE
Is there a way to permanently shift stderr

### DISCUSSION THREAD

**Discussion by natgeo-wong** [fonsp/Pluto.jl#2199] [Questions about using Pluto]
Hi! I would prefer the `stderr` (NOT just `stdout`) outputs to be shifted permanently to the terminal.  Is there a way to do this?

**Comment by disberd** [comment]
Do you mean that you would want error messages to be sent to the terminal rather than being captured by Pluto and being shown in the cell output?

**Comment by natgeo-wong** [comment]
I would like it if logging from packages was sent to terminal.

**Reply by manuelbb-upb** [reply]
To re-enable the old behavior globally, you can use `capture_stdout=false` with `Pluto.run()`,  see [Wiki](https://github.com/fonsp/Pluto.jl/wiki/%F0%9F%A7%BE-Logging#writing-to-standard-output).
If I want logs for only part of a cell to be printed in the terminal, I do the following:
```julia
using Logging
with_logger( Logging.ConsoleLogger( PlutoRunner.original_stdout ) ) do
	@info "Info Message
end
```

Is this what you are trying to achieve?
